### PR TITLE
wine-discord-ipc-bridge: unstable-2023-08-09 -> 0.0.3

### PR DESCRIPTION
**Pull request by uku3lig**
simple version bump :)

[release notes](https://github.com/0e4ef622/wine-discord-ipc-bridge/releases/tag/v0.0.3)

## Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#linking-nixos-module-tests-to-a-package) to the relevant packages
- [x] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [24.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2411.section.md) (or backporting [23.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) and [24.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2405.section.md) Release notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

### CODE REVIEW

File: pkgs/by-name/wi/wine-discord-ipc-bridge/package.nix [hunk 1]
```diff
@@ -4,15 +4,15 @@
...
   pname = "wine-discord-ipc-bridge";
-  version = "unstable-2023-08-09";
+  version = "0.0.3";
 
   src = fetchFromGitHub {
     owner = "0e4ef622";
     repo = "wine-discord-ipc-bridge";
-    rev = "f8198c9d52e708143301017a296f7557c4387127";
-    hash = "sha256-tAknITFlG63+gI5cN9SfUIUZkbIq/MgOPoGIcvoNo4Q=";
+    rev = "v${finalAttrs.version}";
```
Comment (Scrumplex) [inline pkgs/by-name/wi/wine-discord-ipc-bridge/package.nix:14]: nit

```suggestion
    rev = "refs/tags/v${finalAttrs.version}";
```

**Review by Scrumplex — APPROVED** [review]

**Comment by NovaViper** [comment]
Is this suppose to build on x86_64 Linux platforms? When I try to install it onto my NixOS install (AMD CPU desktop), I get this error
```
       error: Package ‘wine-discord-ipc-bridge-0.0.3’ in /nix/store/aj5384f4ax3fg64zkxrw0b7daka0dxfr-source/pkgs/by-name/wi/wine-discord-ipc-bridge/package.nix:31 is not available on the requested hostPlatform:
         hostPlatform.config = "x86_64-unknown-linux-gnu"
         package.meta.platforms = [
           "i686-windows"
         ]
         package.meta.badPlatforms = [ ]
       , refusing to evaluate.

       a) To temporarily allow packages that are unsupported for this system, you can use an environment variable
          for a single invocation of the nix tools.

            $ export NIXPKGS_ALLOW_UNSUPPORTED_SYSTEM=1

          Note: When using `nix shell`, `nix build`, `nix develop`, etc with a flake,
                then pass `--impure` in order to allow use of environment variables.

       b) For `nixos-rebuild` you can set
         { nixpkgs.config.allowUnsupportedSystem = true; }
       in configuration.nix to override this.

       c) For `nix-env`, `nix-build`, `nix-shell` or any other Nix command you can add
         { allowUnsupportedSystem = true; }
       to ~/.config/nixpkgs/config.nix.
```

**Comment by Scrumplex** [comment]
> Is this suppose to build on x86_64 Linux platforms? When I try to install it onto my NixOS install (AMD CPU desktop), I get this error

You are supposed to build this using mingw, as it will be running in Wine. You can do so by building `pkgsCross.mingw32.wine-discord-ipc-bridge`

**Comment by NovaViper** [comment]
👍🏾 Ah ok! I was used to building the package from the nix-gaming repo; thanks

**Pull request merged**